web/styles/zulip.css

```diff
@@ -1871,9 +1871,11 @@ div.focused-message-list {
     right: 8px;
     border-radius: 4px;
 
-    /* For FontAwesome icons used in place of images for some users. */
-    font-size: 19px;
-    text-align: center;
+    /* For FontAwesome icons and zulip icons used in place of images for some users. */
+    &.typeahead-icon {
+        font-size: 19px;
+        text-align: center;
+    }
```

Comment:
Instead of removing this CSS `.typeahead-image` class without `typeahead-icon`, let just add this css for `typeahead-icon` classes.

Reply:
> Instead of removing this CSS `.typeahead-image` class without `typeahead-icon`, let just add this css for `typeahead-icon` classes.

Did you mean something like this : 

```CSS
.typeahead-image {
    display: inline-block;
    height: 21px;
    width: 21px;
    position: relative;
    margin-top: -4px;
    vertical-align: middle;
    top: 2px;
    right: 8px;
    border-radius: 4px;

    &.no-presence-circle {
        margin-left: 9px;
        top: 3px;
    }
}

/* For FontAwesome icons and zulip icons used in place of images for some users. */
.typeahead-icon {
   font-size: 19px;
   text-align: center;
}
```




> I wonder if we event need to introduce a new `typeahead-icon` iclass here. We can just apply this CSS specifically to `zulip-icon-triple-users` and remove changes related to adding `typeahead-icon ` to other `typeahead-image` elements below.

We require this CSS to be applicable to both ```zulip-icons``` and ```FontAwesome icons```. If we specifically tailor it for ```zulip-icon-triple-users```, FontAwesome icons won't utilize this CSS.

Reply:
I mean this:


```

.typeahead-image {
    display: inline-block;
    height: 21px;
    width: 21px;
    position: relative;
    margin-top: -4px;
    vertical-align: middle;
    top: 2px;
    right: 8px;
    border-radius: 4px;

    /* For FontAwesome icons and zulip icons used in place of images for some users. */
    font-size: 19px;
    text-align: center;
    
    &.zulip-icon-triple-users {
        font-size: 19px;
    }

    &.no-presence-circle {
        margin-left: 9px;
        top: 3px;
    }
}
```

Reply:
Got it. Made the required changes.